test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/Output/DesignTime/Inject.cs

```diff
@@ -34,8 +34,12 @@ public class _TestFiles_Input_Inject_cshtml : global::Microsoft.AspNetCore.Mvc.R
         {
         }
         #pragma warning restore 1998
-        [global::Microsoft.AspNetCore.Mvc.Razor.Internal.RazorInjectAttribute]
-        public MyApp MyPropertyName { get; private set; }
+#line 2 "/TestFiles/Input/Inject.cshtml"
```

Comment:
These started showing up now that we know where they came from. I think some code we wrote in `@inject` copies these locations over to the new node.